crates/proto/src/rr/record_data.rs

```diff
@@ -1290,9 +1291,11 @@ mod tests {
     }
 
     #[test]
+    #[allow(clippy::unused_enumerate_index)]
```

Comment:
How about `cfg_attr(not(feature = "std"), expect(clippy::unused_enumerate_index))`?